README.md

```diff
@@ -195,6 +195,13 @@ If you apply the diff (i.e. update and save the demo.jl), JET will automatically
 No errors !
 ```
 
+## Limitations
+
+JET explores the functions you call directly as well as their *inferrable* callees. However, if the argument-types for a call cannot be inferred, JET does not analyze the callee. Consequently, a report of `No errors!` does not imply that your entire codebase is free of errors.
```

Comment:
JET can also change `No errors!` to `Potential errors!` when it finds uninferrable callees (though JET doesn't do so by default since it will appear too noisy). I won't change this default behavior (i.e. keep the default mode unsound) in the (near) future, and so it's great that this section clarifies the limitation.

Reply:
I agree. Maybe make it slightly less joyous? How about "No errors detected" without the exclamation point, which to me implies celebration that the code has been certified as free of errors (a stronger guarantee than is feasible).

Reply:
"No errors detected" seems more solid, thanks for the suggestion!